contracts/SinglePlayerCommit.sol

```diff
@@ -163,84 +149,72 @@ contract SinglePlayerCommit is ChainlinkClient, Ownable {
     /// @notice Create commitment, store on-chain and emit event
     /// @param _activityKey Keccak256 hashed, encoded name of activity
     /// @param _goalValue Distance of activity as goal
-    /// @param _startTime Starttime of commitment, also used for endTime
+    /// @param _daysToStart Starttime of commitment, also used for endTime
```

Comment:
I think we should generalize this to `_timeToStart` for greater precision. For example, we should be able to handle scenarios when a user wants to start in a few hours, or some number of hours and minutes.

Also, lets update the comment to reflect the new param.